travel-event-host-app/src/components/searchSection/searchSection.tsx

```diff
@@ -40,8 +55,8 @@ export default function SearchSection({ keyword }: { keyword: string }) {
 
       <FilterBox
         filterBoxIsOpen={filterBoxIsOpen}
-        setCategories={setCategories}
-        categories={categories}
+        setCategories={setCategoryCheckboxState}
+        categories={categoryCheckboxState}
```

Comment:
We should probably rename this prop to be more descriptive of what it is doing (keeping track of the checked state of the categories)